src/components/HostStatusBadge.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { HostStatus } from "types/host";
+import Badge, { Variant } from "@leafygreen-ui/badge";
+import styled from "@emotion/styled";
+
+interface Props {
+  status: string;
+}
+
+export const HostStatusBadge: React.FC<Props> = ({ status }) => (
+  <StyledBadge variant={statusToBadgeVariant[status as HostStatus]}>
```

Comment:
you will not need the `as HostStatus` when you make status in Props of type `HostStatus`